src/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -78,11 +82,16 @@ if(ENABLE_GAPI_PREPROCESSING)
     target_compile_definitions(${TARGET_NAME} PUBLIC ENABLE_GAPI_PREPROCESSING)
 endif()
 
+add_custom_target(genexdebug COMMAND ${CMAKE_COMMAND} -E echo "$<$<BOOL:${OpenCV_FOUND}>:HAS_OPEN_CV>")
+
+target_compile_definitions(${TARGET_NAME} PUBLIC $<$<BOOL:${OpenCV_FOUND}>:HAS_OPEN_CV>)
+target_link_libraries(${TARGET_NAME} PUBLIC $<$<BOOL:${OpenCV_FOUND}>:opencv_imgproc opencv_core>)
```

Comment:
Do we need OpenCV here at all? I suppose extracting of Y channel can be checked without external dependencies, because we don't have computations here.

Reply:
I based on example test and tests in [prepreocess_opencv.cpp](https://github.com/openvinotoolkit/openvino/blob/09446e1ce2ba273d2d6c1562e82f0ba4feaa687e/src/plugins/template/tests/functional/subgraph_reference/preprocess_opencv.cpp) to have reference values generated in similar way as in these tests.

But is true that for `to grey` conversion there is no need for computation. If that will be OK from test perspective I can re-factor tests and remove external dependencies.

Reply:
Dependency removed.

Reply:
thanks!